### ISSUE TITLE
Some index sets do not contain mapping for gl2_second_sort_field.

### ISSUE THREAD

**Issue by luk-kaminski** [Graylog2/graylog2-server#20817]
## Expected Behavior
All  index sets contain mapping for `gl2_second_sort_field`.

## Current Behavior
Some index sets (i.e. events) do not contain mapping for `gl2_second_sort_field`.
Because of that, tie-breaker is missing in search-after pagination.
See https://github.com/Graylog2/support/issues/120

## Possible Solution
Add missing mapping.

## Steps to Reproduce (for bugs)
See https://github.com/Graylog2/support/issues/120
(Have a lot of of data, so that many messages end up having the same timestamp.)

## Context
Support issue.

## Your Environment
See https://github.com/Graylog2/support/issues/120

**Comment by tellistone** [comment]
From relavent thread;

there are 3 different scenarios worth mentioning.

Scenario 1:
Absolute time range is used, with both from and to in the past.
Outcome: counts are the same.
Explanation: Even if new data come in between search execution and CSV export execution, they do not match the timerange and are not considered by the export.

Scenario 2:
Absolute time range is used, with to being a timestamp in the future (let's say a date in 2042).
Outcome: count in export may be bigger than in search.
Explanation: It seems correct, as after search execution new data is coming, which are exported when export is invoked.

Scenario 3:
Relative range is used.
Outcome: counts can be completely different.
Explanation: last 5 minutes is a different time window during search execution and export execution happening a few moments later.

We could say there is no bug at all and explain that it is intentional that export and search are performed in different moments in time, meaning that data can be different between their executions in some timerange scenarios (namely 2 and 3).
We could also question the current approach and try to implement a way to ensure consistency of data between search and export ("freezing" timestamp under-the-hood, replacing relative with matching absolute timerange under-the-hood...), although it may have some consequences as well and may require discussion with our Tech Lead and Product.

To sum up:

 The scenarios described seem to work as designed. We can accept them/explain them or start talking about a need to change them.

Problems noticed by customer are most probably related to the fact that in some scenarios GL can duplicate messages (i.e. DW restore scenario, the same message in two different streams which use separate index sets scenario). Because of technical limitations export can miss some of those duplicates when exporting. @bernd knows about this limitations and suggests that We probably need to add some warnings to the export feature that message counts can be different due to our handling of duplicates during the export.

**Comment by tellistone** [comment]
Adding UI warnings in the appropriate places seems best.
